examples/website/brushing/brushing-layers/scatterplot-brushing-layer.js

```diff
@@ -18,30 +18,47 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import {ScatterplotLayer} from 'deck.gl';
-
-import scatterplotVertex from './scatterplot-brushing-layer-vertex.glsl';
-import scatterplotFragment from './scatterplot-brushing-layer-fragment.glsl';
+import {ScatterplotLayer} from '@deck.gl/layers';
+import brushingShaderModule from './brushing-shader-module';
 
 const defaultProps = {
-  ...ScatterplotLayer.defaultProps,
   enableBrushing: true,
-  // show point only if source is in brush
+  // show point only if target is in brush
   brushTarget: false,
   // brush radius in meters
   brushRadius: 100000,
-  mousePosition: [0, 0],
+  mousePosition: null,
   getTargetPosition: d => d.target,
   radiusMinPixels: 0
 };
 
 export default class ScatterplotBrushingLayer extends ScatterplotLayer {
   getShaders() {
-    // get customized shaders
-    return Object.assign({}, super.getShaders(), {
-      vs: scatterplotVertex,
-      fs: scatterplotFragment
-    });
+    const shaders = super.getShaders();
+
+    shaders.modules.push(brushingShaderModule);
+
+    shaders.inject = {
+      'vs:#decl': `
+attribute vec3 instanceTargetPositions;
+
+uniform bool enableBrushing;
```

Comment:
Many shader modules have an enabled uniform, maybe this can be moved into the module?